README.md

```diff
@@ -2,6 +2,18 @@
 
 Substra end-to-end (e2e) tests
 
+# A few words regarding tests design
+
+- Must be fast to run
```

Comment:
We could prefix the list with something like

```
When adding or modifying tests, please follow these guidelines.

Each new test:
```

Then each bullet point can start with _should_, _must_, etc. What do you think?

Reply:
Looks like a good idea! Thanks.

Reply:
This is not as easy as it sounds, as some guidelines are for a single test and others for the tests suite.